### ISSUE TITLE
[🐛 Bug]: security vulnerabilities on selenium/standalone-chrome:4.22.0-20240621

### ISSUE THREAD

**Issue by Lunar-Edward** [SeleniumHQ/docker-selenium#2304]
### What happened?

There are some security vulnerabilities on selenium/standalone-chrome:4.22.0-20240621
CVE-2023-36632
CVE-2021-28861
CVE-2023-27043

### Command used to start Selenium Grid with Docker (or Kubernetes)

```shell
Currenty using selenium/standalone-chrome:4.22.0-20240621 with docker
```

### Relevant log output

```shell
none
```

### Operating System

Centos

### Docker Selenium version (image tag)

standalone-chrome:4.22.0-20240621

**Comment by VietND96** [comment]
@Lunar-Edward, Can you please have a scan on image tag `selenium/standalone-chrome:nightly` and confirm CVEs are resolved?
That would be the upcoming release for this month.

**Comment by Lunar-Edward** [comment]
> @Lunar-Edward, Can you please have a scan on image tag `selenium/standalone-chrome:nightly` and confirm CVEs are resolved? That would be the upcoming release for this month.

Thank you very much! CVEs are resolved in image tag `selenium/standalone-chrome:nightly`.

**Comment by VietND96** [comment]
Please verify the latest image tag released `4.23.0-20240727`
